datafusion/physical-expr/Cargo.toml

```diff
@@ -61,6 +61,7 @@ chrono = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-execution = { workspace = true }
 datafusion-expr = { workspace = true }
+datafusion-functions = { workspace = true }
```

Comment:
Can it be moved into `dev-dependencies`? I didn't verify and just be curious.

Reply:
I will verify it , Thanks

Reply:
I don't think we should have a dependence between datafusion-functions and datafusion-physical-exprs as we are trying to make the crates more indenpendent. I commented below on some ways to avoid this dependency

Reply:
Got it